content/applications/finance/accounting/get_started/shared_accounts.rst

```diff
@@ -0,0 +1,159 @@
+list of companies this account is available for.
+
+.. image:: shared_accounts/shared_accounts_configure_shared_accounts_accounting.png
+   :alt: Adding a company to a shared account in Odoo Accounting
+
+.. _merge_tool:
+
+Merge Tool
+==========
+
+Use cases
+---------
+
+The merging tool that accompanies the new shared accounts features is there to simplify the process
+of creating shared accounts. This tool is particularly useful when you have an **already setup
+multi-company environment** in which you were using separate accounts that you now want to merge.
+
+Steps
+-----
+
+To use the merge tool of the shared accounts feature, you need to select all the companies that
+have an account you want to merge in the company selector in the top right corner of the screen.
+
+.. image:: shared_accounts/shared_accounts_merge_tool_select_companies.png
+   :alt: Selecting all companies that have accounts to be merged.
+
+You can then select all the account that are to be merged, click the :guilabel:`Actions` button,
+and select the :guilabel:`Merge accounts` option in the drop-down.
+
+.. image:: shared_accounts/shared_accounts_merge_tool_merge_action.png
+   :alt: Selecting the accounts to merge and the tool in the options.
+
+You will then enter a confirmation wizard where you can select the account you want to merge and
+confirm the operation. Once all is set, you can click the :guilabel:`Merge` button to complete the
+process.
+
+.. image:: shared_accounts/shared_accounts_merge_tool_confirmation_wizard.png
+   :alt: Confirmation wizard for the Merge Tool of the shared accounts feature.
+
+In the end, the selected accounts will be merged into a single shared account that is available for
+all the selected companies in the same way it would have been if the account had only been created
+on a single company and then configured to be shared like in the :ref:`basic setup
+<setting_up_shared_accounts>`.
+
+.. image:: shared_accounts/shared_accounts_merge_tool_result.png
+   :alt: Result of the Merge Tool of the shared accounts feature.
+
+.. _unmerge_tool:
+
+Unmerge Tool
+============
+
+Use cases
+---------
+
+At the opposite end of the spectrum, the unmerge tool is there to simplify the process of stopping
+to share specific accounts. This tool finds its use when you have shared accounts that you want to
+**dissociate from certain companies** without losing the data that they contain.
+
+Steps
+-----
+
+Using any of the companies that have a shared account, it is possible to select it and unmerge it
+using the :guilabel:`Unmerge account` option in the :guilabel:`Actions` menu.
+
+.. image:: shared_accounts/shared_accounts_unmerge_tool_unmerge_action.png
+   :alt: Selecting an account to unmerge and the tool in the options.
+
+This will open a warning window confirming all the accounts that will be created if the action is
+confirmed using for each of them the code setup for the company it will be created in.
```

Comment:
```suggestion
This will open a warning window confirming all the accounts that will be created.
```
Not need to say too much IMO ; that's implicit anyway, and keeping the sentence simpler makes it easier to understand :)